pkgs/applications/terminal-emulators/kitty/go-modules.nix

```diff
@@ -0,0 +1,29 @@
+{stdenv
+, go
+, git
+, cacert
+, src
+, version
+, vendorHash
+}: stdenv.mkDerivation {
```

Comment:
Why is this not using buildGoModule/buildGoPackage?

Also some whitesapce would not hurt

Reply:
`buildGoModule` an `buildGoPackage` attempts to build the binary. This fails because the kitty executable and a functioning python must be present to generate some of the go code.